cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/sponge/platform/CloudNetSpongeClientPingEventResponsePlayers.java

```diff
@@ -0,0 +1,55 @@
+package de.dytanic.cloudnet.ext.bridge.sponge.platform;
+
+import de.dytanic.cloudnet.ext.bridge.server.BridgeServerHelper;
+import de.dytanic.cloudnet.wrapper.Wrapper;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.event.server.ClientPingServerEvent;
+import org.spongepowered.api.profile.GameProfile;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class CloudNetSpongeClientPingEventResponsePlayers implements ClientPingServerEvent.Response.Players {
+
+    static @NotNull ClientPingServerEvent.Response.Players fromCloudNet() {
+        return new CloudNetSpongeClientPingEventResponsePlayers(
+                Wrapper.getInstance().getCurrentServiceInfoSnapshot().getProperties().getInt("Online-Count"),
+                BridgeServerHelper.getMaxPlayers()
+        );
+    }
+
+    private CloudNetSpongeClientPingEventResponsePlayers(int online, int max) {
+        this.online = online;
+        this.max = max;
+    }
+
+    private int online;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5823302)